src/components/virtual-scroll/virtual-scroll.component.ts

```diff
@@ -54,7 +54,8 @@ const TRACK_BY_IDENTITY_FN = <T>(_index: number, item: T) => item;
         ComponentState.create(VirtualScroll)
     ],
     host: {
-        "[attr.grid-list]": "gridList || null"
+        "[attr.grid-list]": "gridList || null",
+        "[style.display]": "'block'"
```

Comment:
This should be defined under `styles` with the `:host` selector so that it can be more easily overridden in CSS without the use of `!important`.